apps/bestofjs-nextjs/src/app/projects/[slug]/page.tsx

```diff
@@ -29,7 +29,6 @@ export async function generateMetadata({
   return {
     title: project.name,
     description: project.description,
-    metadataBase: new URL(`https://${process.env.VERCEL_URL}`), // to avoid warnings at build time
```

Comment:
No need to specify the same `metadataBase` as values are inherited from the root layout.